### DISCUSSION TITLE
Unable to find a way to seed to a specific Postgres Schema

### DISCUSSION THREAD

**Discussion by Jarrodsz** [prisma/prisma#16533] [Q&A] [answered]
aIm trying to seed a specific client schema to a certain postgres schema.

My postgres prisma db has these schemas:
```
public, org1, org2,org3
```

The public schema I seed with regular prisma seed.ts

**after spending several hours and learning a lot im unable to resolve this:**
Im unable to find any way to seed the org1, org2, org3 schema with a custom seed file.

My prisma folder looks like this ( including all prisma files ) 

```
prisma/public/seed.ts
prisma/workspace/seed.ts
```

So basically I would want to do something like:

```
prisma db seed --seed_file=/prisma/workspace/seed.ts schema=org1 or org2/org3
```

Spend several hours on how to fix this problem but im unable to find a solution.
Could someone please give me some pointers on how to solve this?

**What I tried*** 
This seems the most logic way of resolving this but won't work:

```
postgresql://jolex:postgres@localhost:5432/core?schema=org2 npx prisma db seed --schema ./prisma/workspace/schema.prisma
```

**Comment by nurul3101** [comment]
Hello @Jarrodsz 👋 
In your seed file can you pass a connection string with a schema option? 
So it should look something like this:
seed.ts
```ts
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient(
  {
    datasources:{
      db:{
        url:'postgresql://postgres:postgres@localhost:5432/postgres?schema=public'
      }
    }
  }
);

// Rest of the operations
```

Can you try this and check if it works for you?

**Comment by janpio** [comment, marked as answer]
Do you want to see all database schemas with the same Prisma schema?
Then you really just need to use an env var for the connection string, and then replace that before you run `db seed`.

**Reply by Jarrodsz** [reply]
We found it out thank you. was also missing client definition in prisma.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
